docs/make.jl

```diff
@@ -0,0 +1,13 @@
+using Documenter, Interpolations
+makedocs(
+sitename="Interpolations.jl",
+modules=[Interpolations],
```

Comment:
Also add

```julia
format=Documenter.HTML(prettyurls = get(ENV, "CI", nothing)=="true"),
```

so that you can build and inspect documentation locally.

Reply:
Changed. I had to change the Documenter version to 0.21 to use this syntax.